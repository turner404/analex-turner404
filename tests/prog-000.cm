int add(int a, int b){
  return a + b;
}

int main(void){
  int x, y, result;

  x = 5;
  y = 8;
  result = add(x, y);

  output(result);
  return 0;
}
