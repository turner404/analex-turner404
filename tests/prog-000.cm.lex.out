INT
ID
LPAREN
INT
ID
COMMA
INT
ID
RPAREN
LBRACES
RETURN
ID
PLUS
ID
SEMICOLON
RBRACES
INT
ID
COMMA
INT
ID
COMMA
INT
ID
SEMICOLON
ID
ATTRIBUTION
NUMBER
SEMICOLON
ID
ATTRIBUTION
NUMBER
SEMICOLON
ID
ATTRIBUTION
ID
LPAREN
ID
COMMA
ID
RPAREN
SEMICOLON
ID
LPAREN
ID
RPAREN
SEMICOLON
RETURN
NUMBER
SEMICOLON
RBRACES
